index.html

```diff
@@ -60,6 +60,175 @@ <h3 class="H-title">I'am a Software Developer</h3>
                   </ul>
                 </div>
               </section>
+
+              <!--Projects section-->
+              <section id="projects">
+                <div id="h2-container">
+                  <h2 id="h2-projects">My Recent <span>Works</span></h2>
+                  <div id="h2-div"></div>
+                </div>
+                <div id="portfolio-grid">
+                  <div id="main-project">
+                    <div id="main-image-div">
+                      <img
+                        src="./images/Img.png"
+                        alt="project-works"
+                        id="project-works"
+                        data-img="image"
+                      />
+                    </div>
+                    <div id="project-content">
+                      <h3 id="project-h3" data-title="title">Multi-Post Stories</h3>
+                      <p id="content-p">
+                        A daily selection of privately personalized reads; no accounts or
+                        sign-ups required. has been the industry's standard dummy text
+                        ever since the 1500s, when an unknown printer took a standard
+                        dummy text. A daily selection of privately personalized reads; no
+                        accounts or sign-ups required.
+                      </p>
+                      <div class="language-container" data-language="language-list">
+                        <div class="box"><span>css</span></div>
+                        <div class="box"><span>html</span></div>
+                        <div class="box"><span>bootstrap</span></div>
+                        <div class="box"><span>Ruby</span></div>
+                      </div>
```

Comment:
- [ ] Please wrap the technology in a `ul` tag and each should be in a `li tag`.